### ISSUE TITLE
Client.login callback not working

### ISSUE THREAD

**Issue by Desuvader** [hydrabolt/discord.js#105]
Callback seems to be broken. Logs nothing.
```javascript
var Discord = require('discord.js');
var bot = new Discord.Client();
bot.login('email', 'password', function(err, token) {
  if (err) {
    console.log(err);
  }
  console.log('Token below');
  console.log(token);
});
```

**Comment by hydrabolt** [comment]
Apparently fixed in 79990df5b2065e63665c0513493b933e58c075b4

**Comment by abalabahaha** [comment]
Still getting reports, more testing to come.
